index-v4.yaml

```diff
@@ -87,5 +87,11 @@ distributions:
     distribution_status: active
     distribution_type: ros1
     python_version: 3
+  rolling:
+    distribution: [rolling/distribution.yaml]
+    distribution_cache: http://repo.ros2.org/rosdistro_cache/rolling-cache.yaml.gz
```

Comment:
Reminder that this needs to be created at some point.

Reply:
https://github.com/ros2/ros_buildfarm_config/pull/103